data/matchingData/return.json

```diff
@@ -6,53 +6,45 @@
   "type": "return",
   "email": "test@example.com",
   "uuid": "BLACc1363964",
-  "products": [{
-    "salesOfferPackages": [{
-      "name": "onboard sales offer package",
-      "description": "this sales offer package explains how to buy the ticket onboard",
-      "purchaseLocations": [
-        "onBoard"
-      ],
-      "paymentMethods": [
-        "cash",
-        "debitCard",
-        "creditCard"
-      ],
-      "ticketFormats": [
-        "paperTicket"
-      ]
-    },
+  "products": [
     {
-      "name": "mobile app offer package",
-      "description": "this sales offer package explains how to buy the ticket on a mobile app",
-      "purchaseLocations": [
-        "mobileDevice"
-      ],
-      "paymentMethods": [
-        "debitCard",
-        "creditCard",
-        "mobilePhone"
-      ],
-      "ticketFormats": [
-        "mobileApp"
-      ]
-    },
-    {
-      "name": "online offer package",
-      "description": "this sales offer package explains how to buy the ticket on a mobile app",
-      "purchaseLocations": [
-        "online"
-      ],
-      "paymentMethods": [
-        "debitCard",
-        "creditCard",
-        "directDebit"
-      ],
-      "ticketFormats": [
-        "paperTicket"
+      "salesOfferPackages": [
+        {
+          "name": "Onboard (cash)",
+          "description": "Purchasable on board the bus, with cash, as a paper ticket.",
+          "purchaseLocations": ["onBoard"],
+          "paymentMethods": ["cash"],
+          "ticketFormats": ["paperTicket"]
+        },
+        {
+          "name": "Onboard (contactless)",
+          "description": "Purchasable on board the bus, with a contactless card or device, as a paper ticket.",
+          "purchaseLocations": ["onBoard"],
+          "paymentMethods": ["contactlessPaymentCard"],
+          "ticketFormats": ["paperTicket"]
+        },
+        {
+          "name": "Online (smart card)",
+          "description": "Purchasable online, with a debit/credit card or direct debit transaction, on a smart card or similar.",
+          "purchaseLocations": ["online"],
```

Comment:
Should we have some examples with multiple purchase locations?

Reply:
Great idea - I've now added a SOP to each matching data type which has all available options selected